src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,37 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
```

Comment:
In the context of a small application, an import with the name `"cn"` is possible, but in larger applications, where there may be many imports, it is better not to shorten the names to unreadable ones, but to use longer but more obvious names (`import classNames from 'classnames'`).